### ISSUE TITLE
Unable to view the profiles of the contributors when placed at the top part of their icons.

### ISSUE THREAD

**Issue by Abhiram-Gullapalli** [smaranjitghose/indokhaadyam#73]
In the contributions page when trying to view the GitHub profiles until unless the cursor is bought to the very end of their icon, the icon is not being responsive.
Hope it makes some sense, else I'll try adding a screenshot regarding the issue.

**Comment by anushbhatia** [comment]
![image](https://user-images.githubusercontent.com/40017559/157069013-fa757d63-f539-459b-aadd-58c7fc590971.png)
you mean these icons are on extreme left?

**Comment by Abhiram-Gullapalli** [comment]
Exactly, on the very first contributor, notice the yellow thing at the bottom of their profile.
The link to their GitHub profile is only being referred to when placed over the yellow portion and not anywhere above it

**Comment by Abhiram-Gullapalli** [comment]
Also if possible can help me out by explaining to me how to access the index.html file with the help of npm @anushbhatia

**Comment by anushbhatia** [comment]
run npm install
then run npm start 
you will be able to navigate to teams page and access the website form there
